build.sh

```diff
@@ -2,7 +2,7 @@
 repoFolder="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $repoFolder
 
-koreBuildZip="https://github.com/aspnet/KoreBuild/archive/1.0.0.zip"
+koreBuildZip="https://github.com/aspnet/KoreBuild/archive/feature/msbuild.zip"
```

Comment:
This file must also be updated: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Extensions/pull/63/files#diff-0b83f9dedf40d7356e5ca147a077acb4